### PR TITLE
Display nav graph in robofleet visualization

### PR DESCRIPTION
**Pull request by kvsikand**
Add the ability to display the navigation graph in the interface, in addition to the vector graph

### CODE REVIEW

**Comment by kvsikand** [comment]
Note: Once the nav map updates to v2, I'll need to update the map parsing here

File: src/components/Localization2DViewer.tsx [hunk 1]
```diff
@@ -40,7 +45,75 @@ export default function Localization2DViewer(props: {
...
+          if (neigbhor > node_id) {
+            edges.push([
+              node.x,
+              node.y,
+              nodes.get(neigbhor)!.x,
+              nodes.get(neigbhor)!.y,
+            ]);
+          }
+        });
+      }
```
Comment (loganzartman) [inline src/components/Localization2DViewer.tsx:106]: what do you think about pulling this out into like a `parseNavGraph(text) -> {nodes, edges}` in another module?

**Review by loganzartman — APPROVED** [review]
One comment, but lgtm regardless

**Comment by kvsikand** [comment]
Git seems very confused by my rebasing...closing this and making it on a new branch

**Pull request closed without merging**